backend/src/database/seeds/08_picker_options.js

```diff
@@ -132,6 +132,112 @@ exports.seed = function (knex) {
             },
           },
         },
+        {
+          id: 6,
+          name: "project_status",
+          title: "Project Status",
+          description: "The status of the project.",
+          form_name: "Project",
+          form_section: "Project Registration",
+          form_tab: "Project Details",
+          definition: {
+            new_request: {
+              type: "string",
+              title: "New Request",
+            },
+            active: {
+              type: "string",
+              title: "Active",
+            },
+            cancelled: {
+              type: "string",
+              title: "Cancelled",
+            },
+            complete: {
+              type: "string",
+              title: "Complete",
+            },
+          },
+        },
+        {
+          id: 7,
+          name: "portfolio_name",
+          title: "Portfolio Name",
+          description: "Portfolio",
+          form_name: "Project",
+          form_section: "Project Registration",
+          form_tab: "Project Details",
+          definition: {
+            analytics: { type: "string", title: "Analytics", abr: "ANA" },
+            business_communication_solutions: {
+              type: "string",
+              title: "Business & Communication Soltuions",
+              abr: "BCS",
+            },
+            delivery_management_services: {
+              type: "string",
+              title: "Delivery Management Services",
+              abr: "DMS",
+            },
+            enterprise_data_services: {
+              type: "string",
+              title: "Enterprise Data Services",
+              abr: "EDS",
+            },
+            divsional_operations: { type: "string", title: "Divsional Operations", abr: "DO" },
+          },
+        },
+        {
+          id: 8,
```

Comment:
This was not suppose to be created as a picker, mostly because this might be attached to something else (ie might be a query of all projects and fiscals to get a distinct list) also if we do this, we will be updating this picker option every year.